docs/tutorial-model-and-deploy.md

```diff
@@ -0,0 +1,188 @@
+# Tutorial: Model and Deploy an Application with OpenTOSCA
+
+Terms in *cursive* can be found in the [glossary](tosca-glossary.md).
+
+**Requierments:**
+
+ *  GIT (optionally GIT-LFS)
+ *  Docker & Docker Compose
+ *  ca. 6GB RAM
```

Comment:
Update this line to something like at least 6GB of RAM only for Docker to show that we need this much for the ecosystem.

Reply:
Add the links to the installation guides: 
[1]          https://docs.docker.com/get-docker/
[2]          https://docs.docker.com/compose/install/ 
[3]          https://git-scm.com/book/en/v2/Getting-Started-Installing-Git